sklearn/tests/test_grid_search.py

```diff
@@ -161,6 +168,11 @@ def test_grid_search():
     grid_search.decision_function(X)
     grid_search.transform(X)
 
+    # Test the transform operations
+    transformed = grid_search.transform(X)
+    reverse_transformed = grid_search.inverse_transform(transformed)
+    assert_equal(np.array_equal(reverse_transformed, X), True)
```

Comment:
You have to use `assert_array_equal()` instead of `assert_equal(np.array_equal())`

Tips: when you want to compare with `True`, there is `assert_true` `sklearn.utils.testing`